File: logica-de-programacao/secao-9/exercicios/crescente.c
#include <stdio.h>


void limpar_entrada() {
    char c;
    while ((c = getchar()) != '\n' && c != EOF) {}
}


int main(){

    int n1, n2;
    
    printf("Digite dois numeros: \n");
    scanf("%i", &n1);
    limpar_entrada();
    scanf("%i", &n2);

    while (n1 != n2)
    {
        if (n1 < n2){
            printf("CRESCENTE!!\n");
        }
        else{
            printf("DECRESCENTE!\n");
        }

        printf("Digite dois numeros: \n");
        scanf("%i", &n1);
        limpar_entrada();
        scanf("%i", &n2);
    }
    
    
    return 0;
}

File: logica-de-programacao/secao-13/exercicios/crescente.py
n1 = 0
n2 = 1

while n1 != n2:
    print("Digite dois numeros: ")
    n1 = int(input())
    n2 = int(input())

    if n1 < n2:
        print("CRESCENTE!!")
    elif n2 < n1:
        print("DECRESCENTE!!")
